### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Nikita7795** [vercel/vercel#5310] [Help] [answered]
Error:This user account is blocked. Contact support@vercel.com for more information.

why does he write like that?

**Comment by paulogdm** [comment, marked as answer]
Hello,

As the message is showing, this type of issue can only be resolved in our support channel.

**Reply by itxve** [reply]
So am I ,I Want to know How to solve this problem ？   send email ?

**Comment by balanceeco2020** [comment]
这个问题应该怎么解决

**Reply by miniheni** [reply]
同问？
